core/src/main/java/com/klaytn/caver/utils/Utils.java

```diff
@@ -34,7 +34,7 @@ public static boolean isValidAddress(String input) {
         return cleanInput.length() == LENGTH_ADDRESS_String && HEX_STRING.matcher(cleanInput).matches();
     }
 
-    public static boolean isKlaytnWalletKeyFormat(String key) {
+    public static boolean isKlaytnKey(String key) {
```

Comment:
How about changing to `isKlaytnWalletKey`???

Reply:
Ok, I'll change it